src/poc/miner_poc_denylist.erl

```diff
@@ -0,0 +1,184 @@
+                                                                                {error, RenameReason} ->
+                                                                                    lager:notice("failed to rename ~p to ~p: ~p", [TmpDenyFile, DenyFile, RenameReason])
+                                                                            end;
+                                                                        {error, WriteReason} ->
+                                                                            lager:notice("failed to write denyfile ~p to disk ~p", [TmpDenyFile, WriteReason])
+                                                                    end,
+                                                                    ok = persistent_term:put(?MODULE, Filter),
+                                                                    {noreply, schedule_check(State#state{version=Serial})};
+                                                                {error, Reason} ->
+                                                                    lager:notice("failed to deserialize denylist from disk: ~p", [Reason]),
+                                                                    {noreply, schedule_check(State)}
+                                                            end;
+                                                        false ->
+                                                            lager:notice("failed to verify signature on denylist"),
+                                                            {noreply, schedule_check(State)}
+                                                    end;
+                                                _ ->
+                                                    lager:notice("unrecognized or corrupt denylist"),
+                                                    {noreply, schedule_check(State)}
+                                            end;
+                                        ZipDownloadOther ->
+                                            lager:notice("failed to download zip file release ~p : ~p", [ZipURL, ZipDownloadOther]),
+                                            {noreply, schedule_check(State)}
+                                    end
+                            end
+                    end
+            catch
+                _:_ ->
+                    lager:notice("failed to decode github release json: ~p", [Body])
+            end;
+        {ok,{{_,304,"Not Modified"}, _}} ->
+            schedule_check(State0);
+        OtherHttpResult ->
+            lager:notice("failed to fetch github release info ~p", [OtherHttpResult]),
+            schedule_check(State0)
+    end;
+handle_info(Msg, State) ->
+    lager:info("unhandled info msg ~p", [Msg]),
+    {noreply, State}.
+
+handle_cast(Msg, State) ->
+    lager:info("unhandled cast msg ~p", [Msg]),
+    {noreply, State}.
+
+handle_call(Msg, _From, State) ->
```

Comment:
looks like you haven't handled the two calls in the api yet

Reply:
No, not yet, was going to do that later, maybe, so we can gossip this file around.